__tests__/nc-news-test.test.js

```diff
@@ -53,6 +53,7 @@ describe("GET /api/articles/:article_id", () => {
           votes: 100,
           article_img_url:
             "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+          comment_count: "11",
```

Comment:
You should write a new test for the comment count rather than refactor a previous test